pkg/ibm_plugin/server/plugin_manual_test.go

```diff
@@ -0,0 +1,300 @@
+//go:build ibm
```

Comment:
I am not sure if we need manual test to be committed. The right test for manual testing should be using CLI, I see that it divulges from actual flow. For instance instanceID is supposed to be `/resourcegroup/<resID>/zone/<zone>/instance/<instanceID> `and not `/resourcegroup/<resID>/zone/<zone>/instance/<instanceName>`.

Reply:
Yes, the manual tests aren't up to date, but I do not agree with your statement. Programmatic tests are much more useful for debugging purposes, with greater level of control over input, and ease of use from IDEs' integrated debugger.
I will remove the file for the time being.